core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/GraalVM.java

```diff
@@ -67,7 +66,7 @@ static Version parse(List<String> lines) {
 
                 String vendorVersion = secondMatcher.group(VENDOR_VERSION_GROUP);
 
-                String graalVersion = graalVersion(javaVersion, v.feature());
+                String graalVersion = graalVersion(javaVersion, v.feature(), v.interim(), v.update());
```

Comment:
I think it will be cleaner if we just pass `v` as an option.

```suggestion
                String graalVersion = graalVersion(javaVersion, v);
```

Reply:
Makes sense.